layouts/default.vue

```diff
@@ -147,14 +147,19 @@ export default Vue.extend({
     }
   },
   beforeMount() {
+    this.$nuxt.$loading.start();
```

Comment:
With the change proposed above (ie going back to Nuxt's original behavior), this is not needed.